docker/build-tools/Dockerfile

```diff
@@ -29,17 +32,19 @@ RUN mkdir -p /usr/bin /usr/include
 WORKDIR /tmp
 ADD https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protoc-${PROTOBUF_VERSION}-linux-x86_64.zip /tmp/
 RUN unzip /tmp/protoc-${PROTOBUF_VERSION}-linux-x86_64.zip
-RUN cp -aR /tmp/bin/protoc /usr/bin
 
 # Install necessary protoc includes
-RUN mkdir -p /usr/include/protobuf/google/protobuf /usr/include/protobuf/google/rpc /usr/include/protobuf/gogoproto
+RUN mkdir -p ${OUTDIR}/usr/include/protobuf/google/protobuf
```

Comment:
Best practice is to combine those either in a single command, or a string with &&